www/js/metrics.js

```diff
@@ -844,11 +844,11 @@ angular.module('emission.main.metrics',['nvd3', 'emission.services', 'ionic-date
                 // Here, we check if the string is all upper case by
                 // converting it to upper case and seeing if it is changed
                 if (field == field.toUpperCase()) {
-                    if (field === "WALKING" || field === "RUNNING") {
-                      fieldPhone = "ON_FOOT";
-                    } else {
+                    // if (field === "WALKING" || field === "RUNNING") {
+                    //   fieldPhone = "ON_FOOT";
```

Comment:
Or I guess you are handing this in the footprint calculations (e.g. `getFootprintForMetrics`). Any reason for the change?

Reply:
I knew I didn't want to convert WALKINg to ON_FOOT; but removing/commenting that code was also lazy, because then I can have RUNNING labels which aren't handled in `getFootprintForMetrics`. I think a separate branch should address the mode consistency (i.e. check for and convert certain modes) at some point. 
I will restore this line and avoid addressing the issue piecemeal before I break something :)